### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by ouziel-slama**
- [x] Inject 'unpacked_data' with verbose
- [x] More Detailed InsufficientBTC Error
- [x] tweak RPS logs
- [x] Clean `genapidoc.py` (no need to generate docusaurus doc)
- [x] Add 'quantity_normalized' in issuances endpoints
- [x] Remove 'asset_info' from ASSET_ISSUANCE event

* [x] Double-check the spelling and grammar of all strings, code comments, etc.
* [x] Double-check that all code that needs to be is deterministic
* [x] Add tests to cover any new features
* [x] Ensure that the test suite passes with the proposed changes
* [x] Update the project [release notes](release-notes/)
* [x] Update the project documentation as appropriate with a corresponding Pull Request in the [Documentation repository](https://github.com/CounterpartyXCP/Documentation)

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1730,7 +1730,25 @@ Returns the transactions of a block
...
-                    "btc_amount_normalized": "0"
+                    "btc_amount_normalized": "0",
+                    "unpacked_data": {
+                        "message_type": "issuance",
+                        "message_type_id": 22,
+                        "message_data": {
+                            "asset_id": 75313533584419238,
+                            "asset": "UNNEGOTIABLE",
+                            "subasset_longname": null,
+                            "quantity": 1,
```
Comment (adamkrellenstein) [inline apiary.apib:1741]: shouldn't this be `true`/`false` everywhere?

File: apiary.apib [hunk 2]
```diff
@@ -2339,36 +2357,60 @@ Returns the list of the last ten transactions
...
+                    "data": "0c00000000000000010000000000000000000000000000000000000000000000000a",
                     "supported": true,
-                    "btc_amount_normalized": "0.18"
+                    "btc_amount_normalized": "0",
+                    "unpacked_data": {
+                        "message_type": "dispenser",
+                        "message_type_id": 12,
+                        "message_data": {
+                            "asset": "XCP",
+                            "give_quantity": 0,
```
Comment (adamkrellenstein) [inline apiary.apib:2401]: is it possible to include normalized quantities here too?

File: apiary.apib [hunk 3]
```diff
@@ -2339,36 +2357,60 @@ Returns the list of the last ten transactions
...
+                    "data": "02000000000144d2d2000000000000000100f97c1bb2e33b09b13015b9d386a72982d403fbea",
                     "supported": true,
-                    "btc_amount_normalized": "0"
+                    "btc_amount_normalized": "0",
+                    "unpacked_data": {
+                        "message_type": "enhanced_send",
+                        "message_type_id": 2,
+                        "message_data": {
+                            "asset": "BUPEPE",
+                            "quantity": 1,
```
Comment (adamkrellenstein) [inline apiary.apib:2377]: and `quantity_normalized`?

**Review by adamkrellenstein — CHANGES_REQUESTED** [review]

File: apiary.apib (hunk 2 above)
Comment (ouziel-slama) [reply apiary.apib:2384]: yep! done.

File: apiary.apib (hunk 3 above)
Comment (ouziel-slama) [reply apiary.apib]: done!

**Review by adamkrellenstein — APPROVED** [review]

**Pull request merged**